templates/certified/index.html

```diff
@@ -0,0 +1,721 @@
+{% extends "templates/base.html" %}
+
+{#-
+  TODO replace with upstreamed macro
+  import after https://github.com/canonical/vanilla-framework/pull/5503 is merged
+-#}
+{% from "macros/_vf_tiered-list.jinja" import vf_tiered_list %}
+
+{% block title %}Ubuntu官方认证硬件 {{ title }}{% endblock %}
+
+{% block meta_description %}
+  {{ meta_description }}
+  通过Ubuntu官方认证的硬件，如服务器、笔记本和工作站、IoT物联网设备等等可以获得更好的支持，涵盖最长12年的集成测试和审查，提供开箱即用的体验，无需为繁琐的系统组件驱动程序和兼容性而担忧。
+{% endblock %}
+
+{% block body_class %}
+is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-section--hero">
+      <div class="row--25-75">
+        <div class="col">
+          {{ image(url="https://assets.ubuntu.com/v1/b9b1cbf0-Ubuntu-certified-badge-RGB.svg",
+            alt="Ubuntu 认证硬件徽标",
+            width="90",
+            height="128",
+            hi_def=True,
+            loading="auto"
+          ) | safe }}
+        </div>
+        <div class="col">
+          <h1>Ubuntu 硬件认证计划</h1>
+          <h2>
+            通过广泛的测试和审查过程，<br/>
+            为您确保最佳的Ubuntu使用体验
+          </h2>
+          <p>
+            通过 Ubuntu 官方认证的硬件，如服务器、笔记本和工作站、IoT物联网设备等等可以获得更好的支持，开箱即用的体验，
+            且无需为繁琐的系统组件驱动和兼容性而担忧。
+          </p>
+          <hr class="p-rule--muted" />
+          <a class="p-button--positive" href="/contact">与我们联系</a>
+        </div>
+      </div>
+  </section>
+
+  {%- call(slot) vf_tiered_list(
+    is_description_full_width_on_desktop=false,
+    is_list_full_width_on_tablet=true,
+    is_image_full_width=false
+  ) -%}
+    {%- if slot == 'title' -%}
+      <h2>通过认证的硬件享最佳<br/>Ubuntu体验</h2>
+    {%- endif -%}
+
+    {%- if slot == 'description' -%}
+      <p>预装开箱即用的Ubuntu操作系统，并已通过广泛的可靠性和性能测试</p>
+    {%- endif -%}
+
+    {%- if slot == "image" -%}
+      <div class="p-image-container--16-9 is-cover">
+        {{ image(
+          url="https://assets.ubuntu.com/v1/966f0d9f-laptop.png",
+          alt="",
+          width="1200",
+          height="675",
+          hi_def=True,
+          loading="auto",
+          attrs={"class": "p-image-container__image"}
+        ) | safe }}
+      </div>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_1' -%}
+      <h3 class="p-heading--5">选择值得信赖的硬件</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_1' -%}
+      <p>查看Ubuntu认证机型列表，选择符合您需求的合作伙伴硬件。</p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_2' -%}
+      <h3 class="p-heading--5">降低成本</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_2' -%}
+      <p>可靠的认证硬件可簡化解决方案的实施，降低操作系统开发和支持的成本。</p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_3' -%}
+      <h3 class="p-heading--5">加速产品上市</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_3' -%}
+      <p>利用 Canonical 和硬件合作伙伴的专业知识，快速将您的解决方案推向市场。</p>
+    {%- endif -%}
+  {%- endcall -%}
+
+  <section class="p-strip">
+    <div class="p-strip u-no-padding--top">
```

Comment:
```suggestion
    <div class="p-section">
```
Achieves the same thing